builtin/commands/dposV3_commands.go

```diff
@@ -343,7 +343,7 @@ func UnbondCmdV3() *cobra.Command {
 	return &cobra.Command{
 		Use:   "unbond_v3 [validator address] [amount]",
```

Comment:
update tooltip to include index, and lgtm

Reply:
thanks.